pkg/pool-weighted/contracts/WeightedMath.sol

```diff
@@ -376,15 +369,14 @@ library WeightedMath {
 
         // We can now compute how much excess balance is being withdrawn as a result of the virtual swaps, which result
         // in swap fees.
-        uint256 taxablePercentage = normalizedWeight.complement();
 
         // Swap fees are typically charged on 'token in', but there is no 'token in' here, so we apply it
         // to 'token out'. This results in slightly larger price impact. Fees are rounded up.
-        uint256 taxableAmount = amountOutWithoutFee.mulUp(taxablePercentage);
+        uint256 taxableAmount = amountOutWithoutFee.mulUp(normalizedWeight.complement());
         uint256 nonTaxableAmount = amountOutWithoutFee.sub(taxableAmount);
+        uint256 swapFee = taxableAmount.mulUp(swapFeePercentage);
```

Comment:
I would keep it consistent with previous functions and write:

```
uint256 taxableAmountMinusFees = taxableAmount.mulUp(swapFeePercentage.complement());
return nonTaxableAmount.add(taxableAmountMinusFees);
```

Reply:
Can do - we no longer need to isolate the swap fees, which is why it was done that way.